src/jsp/com/cubrid/jsp/impl/SUStatement.java

```diff
@@ -154,6 +155,24 @@ public SUStatement(
         isSensitive = false;
     }
 
+    public SUStatement(SUConnection conn, GetGeneratedKeysInfo info) {
+        type = GET_AUTOINCREMENT_KEYS;
```

Comment:
This and another SUStatement constructors do not set suConn field with the conn argument.
Is it OK?

Reply:
I've fixed. thank you.